torch/csrc/jit/passes/onnx/remove_inplace_ops_for_onnx.cpp

```diff
@@ -606,11 +1009,15 @@ void PrepareInplaceOpsForONNX(const std::shared_ptr<Graph>& graph) {
   PrepareListAppendAndInsertForONNX(graph->block());
 }
 
-void RemoveInplaceOpsForONNX(const std::shared_ptr<Graph>& graph) {
+void RemoveInplaceOpsForONNX(
+    const std::shared_ptr<Graph>& graph,
+    Module* model = nullptr) {
   MutationRemover mr(graph);
   PrepareForRemoveMutations(mr, graph->block());
   RemoveTensorMutation(graph);
   RemoveListMutation(graph);
+  if (model)
+    RegisterInplaceOpAsBlockOutputs(mr, model, graph);
```

Comment:
Would `RemoveTensorMutation` convert ops that needs attention in this pass to outplace before reaching here?

Reply:
Yes, we would need to re-order this for those ops. Will do as the next step.